draft-ietf-quic-recovery.md

```diff
@@ -406,7 +406,7 @@ Pseudocode for OnAlarm follows:
 ~~~
    OnAlarm(acked_packet):
      lost_packets = DetectLostPackets(acked_packet);
```

Comment:
Yes, I figured that it would be OK to change the name of a "local" variable in that context.  I was going to change this, but then I have to change a bunch of other things and it started to cascade.  I'll let you sort that mess out :)